jumanji/environments/routing/mmst/env.py

```diff
@@ -388,43 +400,35 @@ def _state_to_timestep(
             timestep: TimeStep object containing the timestep of the environment.
         """
 
-        observation = self._state_to_observation(state)
         reward = self._reward_fn(state, action, state.nodes_to_connect)
 
         # Update the state now.
         state.finished_agents = self.get_finished_agents(state)
         state.step_count = state.step_count + 1
         extras = self._get_extras(state)
+        observation = self._state_to_observation(state)
 
-        def make_termination_timestep(state: State) -> TimeStep:
+        def make_termination_timestep() -> TimeStep[Observation]:
             return termination(
                 reward=reward,
                 observation=observation,
                 extras=extras,
             )
 
-        def make_transition_timestep(state: State) -> TimeStep:
+        def make_transition_timestep() -> TimeStep[Observation]:
             return transition(
                 reward=reward,
                 observation=observation,
                 extras=extras,
             )
 
-        is_done = state.finished_agents.all()
+        agents_are_done = state.finished_agents.all()
```

Comment:
Why agents in plural if there is only a single agent (indicated by previous line of code)?

Reply:
The state attribute is called `finished_agents`, so I want to improve consistency by calling the `done` an `agents_are_done` variable. Does it make sense?

Reply:
Sounds good 👍